compiler/AST/checkAST.cpp

```diff
@@ -214,8 +216,6 @@ void checkPrimitives()
      case PRIM_ARRAY_SHIFT_BASE_POINTER:
      case PRIM_ARRAY_SET:
      case PRIM_ARRAY_SET_FIRST:
-     case PRIM_ERROR:
-     case PRIM_WARNING:
```

Comment:
Oh, my bad. I misread the diff and thought checkPrimitives() complained for prim_warning/error after resolution.